config/configgrpc/configgrpc.go

```diff
@@ -84,6 +85,10 @@ type GRPCClientSettings struct {
 
 	// The headers associated with gRPC requests.
 	Headers map[string]string `mapstructure:"headers"`
+
+	// Sets the balancer as RoundRobin in grpclb_policy to discover the servers. Default is pick_first
+	// https://github.com/grpc/grpc-go/blob/master/examples/features/load_balancing/README.md
+	useRoundRobin bool `mapstructure:"useRoundRobin"`
```

Comment:
I would use a string here instead of book, just in case other strategies will be supported. Same behavior, if not set use the first pick. What do you think?

Reply:
Changed it from bool to string and validated the same as setting invalid balancer name causes panic.